apps/admin-gui/src/app/facilities/pages/facility-detail-page/facility-services-destinations/facility-services-destinations.component.html

```diff
@@ -15,6 +15,24 @@ <h1 class="page-subtitle">{{'FACILITY_DETAIL.SERVICES_DESTINATIONS.TITLE' | tran
     [disabled]="selected.selected.length === 0"  (click)="removeDestination()">
     {{'FACILITY_DETAIL.SERVICES_DESTINATIONS.REMOVE' | translate}}
   </button>
+  <button
+    *ngIf="!loading && allowAuth"
+    mat-flat-button
+    color="accent"
```

Comment:
I don't really like the style of these buttons. You should usually have only one action and one danger button at most. This really makes it look weird. I would just use a neutral color for these two buttons.